CHANGELOG.md

```diff
@@ -21,6 +21,10 @@
   details.
 - Context modifier no longer overrides an existing token on the context. This allows `buf registry login`
   to properly check the user provided token without the token being overriden by the CLI interceptor.
+- Removed the `buf config init` command in favor of `buf mod init`.
+- Removed the `buf config ls-breaking-rules` command in favor of `buf mod ls-breaking-rules`.
+- Removed the `buf config ls-lint-rules` command in favor of `buf mod ls-lint-rules`.
+- Removed the `buf config migrate-v1beta1` command in favor of `buf beta migrate-v1beta1`.
```

Comment:
This should have been done in the other PR, but could you add a note for adding the `buf beta decode` command in the `CHANGELOG.md`? For reference, see https://github.com/bufbuild/buf/pull/925.